src/tests/ftest/SConscript

```diff
@@ -14,7 +14,7 @@ def scons():
             'daos_perf', 'daos_racer', 'daos_vol',
             'daos_test', 'data', 'fault_domain', 'io', 'ior',
             'mdtest', 'network', 'nvme', 'mpiio',
-            'object', 'osa', 'pool', 'rebuild', 'security',
+            'object', 'osa', 'pool', 'posix', 'rebuild', 'security',
```

Comment:
Do we need a separate 'posix' dir. Posix basically means for us to test dfuse so having the test under 'dfuse' dir should be better than creating another dir.